Client/src/layouts/default.css

```diff
@@ -0,0 +1,24 @@
+.page__container {
```

Comment:
zou je deze willen veranderen in `.page-container`, aangezien we in het project geen BEM hanteren? 
geldt ook voor de regels hieronder.